### PR TITLE
Add Module11 level 06 - 08

### PR DESCRIPTION
**Pull request by dzmitri-litvinov**

### CODE REVIEW

File: autopark/Program.cs [hunk 1]
```diff
@@ -91,10 +91,15 @@ public static void Main(string[] args)
                               "//Level 05//\n" +
                               "////////////");
 
+            Console.WriteLine(new string('*', scrWidth) + '\n');
```
Comment (PushMeNow) [inline autopark/Program.cs]: I see you use same line some times. Move this action in separate method. Keep DRY approach.

File: autopark/Program.cs [hunk 2]
```diff
@@ -103,14 +108,60 @@ public static void Main(string[] args)
...
 
             VehiclesCollection.Sort(new VehicleNameComparer());
-
-            Console.WriteLine(new string('*', scrWidth) + '\n');
+                        
             VehiclesCollection.Print();
+
+            Console.WriteLine("////////////\n" +
+                              "//Level 06//\n" +
+                              "////////////");
```
Comment (PushMeNow) [inline autopark/Program.cs]: Again move it in separate method

File: autopark/SupportClasses/SparePartsDictionary.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    {
+        public Dictionary<string, int> SparePartsDict { get; set; } = new();
+        public string[] SparePartsNames { get; set; }
+
+        public SparePartsDictionary()
+        { }
+
+        public SparePartsDictionary(string inFile)
+        {
+            SparePartsNames = LoadSpareParts(inFile).ToArray();
```
Comment (PushMeNow) [inline autopark/SupportClasses/SparePartsDictionary.cs]: I don't the reason to keep parts in global variable. Transmit result of method LoadSpareParts in CreateDictionary

File: autopark/SupportClasses/SparePartsDictionary.cs [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            CreateDictionary();
+        }
+
+        public IEnumerable<string> LoadSpareParts(string inFile)
```
Comment (PushMeNow) [inline autopark/SupportClasses/SparePartsDictionary.cs]: Remember about incapsulation. User mustn't know about this method.

File: autopark/SupportClasses/SparePartsDictionary.cs [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    }
+                }
+            }
+            else
+            {
+                Console.WriteLine("File does not exist...");
+            }
+        }
+
+        public void CreateDictionary()
```
Comment (PushMeNow) [inline autopark/SupportClasses/SparePartsDictionary.cs]: Again incapsulation

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi man!
I did some comment. Let me know if any question.

File: autopark/Program.cs (hunk 1 above)
Comment (dzmitri-litvinov) [reply autopark/Program.cs]: image
Done

File: autopark/Program.cs (hunk 2 above)
Comment (dzmitri-litvinov) [reply autopark/Program.cs]: image
image
Done

Comment (dzmitri-litvinov) [reply autopark/Program.cs]: I wrote such code to visually separate levels in Main

File: autopark/SupportClasses/SparePartsDictionary.cs (hunk 4 above)
Comment (dzmitri-litvinov) [reply autopark/SupportClasses/SparePartsDictionary.cs]: image
Done

File: autopark/SupportClasses/SparePartsDictionary.cs (hunk 5 above)
Comment (dzmitri-litvinov) [reply autopark/SupportClasses/SparePartsDictionary.cs]: image
Done

File: autopark/SupportClasses/SparePartsDictionary.cs (hunk 3 above)
Comment (dzmitri-litvinov) [reply autopark/SupportClasses/SparePartsDictionary.cs]: image
Shall I overload the method CreateDictionary in such a way, so it has an `IEnumerable<string> parts` argument?

**Review by PushMeNow — APPROVED** [review]
Approved

**Pull request merged**